### ISSUE TITLE
When applying style changes, they work but quickly auto-revert after page finishes loading?

### ISSUE THREAD

**Issue by daytonturner** [johanfrick/custom-sonos-card#113]
**Checklist:**

- [X] I updated to the latest version available
- [X] I cleared the cache of my browser

**Release with the issue:**
v5.20.1

**Last working release (if known):**
n/a

**Browser and Operating System:**
Chrome on MacOS and Windows 10
Safari on MacOS

**Description of problem:**
When overriding styles for the media browser card, the overrides do work, but after loading within 2-8 seconds, the page seems to get an update and the styles revert automatically.  When using the element inspector in chrome, when this happens I can see the elements in side <ha-card> blink blue and get an update.  Not sure what's triggering that, but the refresh is tied to that event occurring.  See attached screen recordings for example: https://imgur.com/5YyEJTh

**Javascript errors shown in the web inspector (if applicable):**
None

**Additional information:**
Here is my card config, including style override settings:

```
type: custom:sonos-media-browser
entityId: media_player.kitchen
mediaBrowserItemsAsList: true
styles:
  button-section:
    border: 0px;
  media-button-thumb:
    background-size: contain;
    height: 100%;
    width: 50px;
  media-buttons:
    maxHeight: 400px;
    overflow: hidden;
  media-browser-header:
    display: none;
  media-button:
    height: 50px;
    border: 0;
    font-size: 32px;
  media-button-title:
    font-size: 25px;
```

**Comment by johanfrick** [comment]
Thanks for the report @daytonturner. Can you check if you can reproduce it in an earlier version of the card? I want to know if this was introduced recently.

**Comment by johanfrick** [comment]
Can you please try again without semicolons? (';'). I don't believe that will work.
```yaml
styles:
  button-section:
    border: 0px
  media-button-thumb:
    background-size: contain
    height: 100%
    width: 50px
  media-buttons:
    maxHeight: 400px
    overflow: hidden
  media-browser-header:
    display: none
  media-button:
    height: 50px
    border: 0
    font-size: 32px
  media-button-title:
    font-size: 25px
```

**Comment by daytonturner** [comment]
I figured it out by setting breakpoints and seeing where it was failing.  There's a section where it seems to search for and split on dashes '-' for different browser compatability.  It also showed me which functions were executing and which elements were being modified, which is how i found the different style setting names..

I changed all instances of 'font-size' to 'fontSize' and 'background-size' to 'backgroundSize' and this has made the issue disappear!
